### ISSUE TITLE
NewsletterBundle: move snippets to Resources directory in bundle for theme handling

### ISSUE THREAD

**Issue by pixeljunkie** [chameleon-system/chameleon-system#850]
**Describe the problem (if applicable)**
The Newsletter Bundle is so old, we had no themes back then. So there is an "install/tocopy" directory in the bundle.
These assets and templates should move to the "Resources" directory and the copy setup step can be removed from the README.md

**Comment by michaelcmichaelis** [comment]
Currently, a simple moving the module and resouce files inside the bundle wouldn't work as the module is not selectable in the template engine if local folder exist.
So a deeper overwork of the template engine (or a nearby function) is necessary first.

**Comment by michaelcmichaelis** [comment]
> Currently, a simple moving the module and resouce files inside the bundle wouldn't work as the module is not selectable in the template engine if local folder exist. So a deeper overwork of the template engine (or a nearby function) is necessary first.

This is not true as the resources have to be on the right places, then you actually can remove any reference files (resources, modules classes) from the project folder.
So you can instantiate a NewsletterSignup/NewsletterSignout module on your preferred pages and select a template of your choice as usual.
